guides/common/modules/proc_installing-webhooks-plugin.adoc

```diff
@@ -1,4 +1,4 @@
-[id="installing-webhooks-plugin_{context}"]
+[id="installing-webhooks_{context}"]
 = Installing Webhooks
```

Comment:
```suggestion
[id="installing-webhooks-plug-in_{context}"]
= Installing Webhooks plug-in
```

It's a Foreman plug-in according to the installer command.

Reply:
I agree. This didn't look right to me. Will fix.